sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -155,6 +161,7 @@ def _add_argparse_args(cls, parser):
 
 def model_pcollection(argv):
   """Creating a PCollection from data in local memory."""
+  # [START model_pcollection]
```

Comment:
Without the definition of options my_options.output will be undefined, which is referenced by WriteToText.

If we don't want to include MyOptions then we should replace WriteToText with some predefined filename. That case the Java & Python code will be somewhat different.